metrics_build/EventHubs_Metrics/sumoclient.js

```diff
@@ -0,0 +1,262 @@
+/*jshint esversion: 6 */
```

Comment:
Is this file needed for metrics collection? If not maybe we can remove.

Reply:
nops it is not needed but in mainindex it's there as a dependency